workers/datasets_based/src/datasets_based/workers/first_rows.py

```diff
@@ -534,7 +535,8 @@ def compute_first_rows_response(
             use_auth_token=use_auth_token,
         )
     except Exception as err:
-        if info.size_in_bytes is None or info.size_in_bytes > max_size_fallback:
+        MAX_SIZE_FALLBACK = 100_000_000
```

Comment:
After having imported warnings, something like:
```suggestion
        if max_size_fallback:
            warnings.warn(<A WARNING MESSAGE HERE>)
        MAX_SIZE_FALLBACK = 100_000_000
```

Reply:
I think you should fix your last commit... :sweat_smile:

Reply:
yes, I know, I just wanted to have you as a co-author in the squashed commit